### ISSUE TITLE
Badlands Death Spawns u in lOCH mODAN

### ISSUE THREAD

**Issue by Bornlesszero** [Wall-core/Everlook-Bugtracker#671]
Uldaman Death Spawns u in lOCH mODAN

**Comment by Daribon** [comment]
The north badlands graveyard wasn't added until patch 1.12.0.
The closest graveyard in earlier patch is Thelsamar which is a neutral graveyard.

Working as intended.

**Comment by FlagFlayer** [comment]
Correct, other than the fact that last I heard Wall wanted graveyards to have 1.12 behaviour from launch. Might need to be adjusted depending on that.
